contrib/IECoreAlembic/include/IECoreAlembic/PrimitiveReader.inl

```diff
@@ -67,9 +69,16 @@ template<typename T, typename GeomParam>
 struct ApplyGeometricInterpretation<IECore::GeometricTypedData<T>, GeomParam>
 {
 
-	static void apply( IECore::GeometricTypedData<T> *data )
+	static void apply( IECore::GeometricTypedData<T> *data, const Alembic::AbcCoreAbstract::PropertyHeader &header )
 	{
-		data->setInterpretation( IGeomParamTraits<GeomParam>::geometricInterpretation() );
+		if( Alembic::AbcGeom::isUV( header ) )
```

Comment:
Out of curiosity, what was ` IGeomParamTraits<GeomParam>::geometricInterpretation()` returning when `Alembic::AbcGeom::isUV( header ) == true` ?

Reply:
It would depend on `GeomParam`, but assuming it was `IV2fGeomParam` (which it almost certainly would be), it would have been returning `Vector`.